src/as-validator-issue-tag.h

```diff
@@ -685,6 +685,11 @@ AsValidatorIssueTag as_validator_issue_tag_list[] =  {
 	  N_("The value set as release type is invalid."),
 	},
 
+	{ "release-property-missing",
```

Comment:
Please split this into `release-version-missing` and e.g. `release-time-missing`, in order to adjust the priorities individually and give a better explanation (especially for the "either `date` or `timestamp` case).